statusHandler/termuiStatusHandler.go

```diff
@@ -0,0 +1,54 @@
+package statusHandler
+
+import "github.com/ElrondNetwork/elrond-go/termuic"
+
+// TermuiStatusHandler will be used when an AppStatusHandler is required, but another one isn't necessary or available
+type TermuiStatusHandler struct {
```

Comment:
I do not understand why this wrapper was created since this just calls *termuic.TermuiConsole's methods.
I strongly suggest that you move here all the logic from *termuic.TermuiConsole and test it as thoroughly as possible (move the metrics initialization, the map that act as a cacher and so on). 
Then let termuic.TermuiConsole do **only** the displaying part. That will be impossible to test in unit tests.

Reply:
All the logic where metrics are saved in sync.Map was moved to TermuiStatusHandler and termuiConsole will receive at creation (constructor parameter) a pointer to the sync.Map and termuiConsole will just read data from map.

Reply:
:+1: